themes/default/content/challenge/holiday-shopping.md

```diff
@@ -0,0 +1,269 @@
+---
+title: "Holiday Shopping"
+layout: challenge/single
+description: |
+    Create a simple shopping list app using AWS Fargate and MongoDB Atlas 
+meta_desc: |
+    Create a simple shopping list app using AWS Fargate and MongoDB Atlas
+meta_image: /images/challenge/challenge_cta.png
+---
+
+## Welcome to the Pulumi Challenge!
+
+<div class="flex flex-wrap md:mt-12">
+  <div>
+    <h3>Holiday Shopping List</h3>
+    <p class="pr-12">
+      The holidays are upon us and mere anarchy has been loosed upon our psyche. More specifically how do you keep track of everything you need to buy for gifts or food?!? Surely some salvation is at hand? Lucky for you, our friend, The Pulumipus, has created a simple to-do list application to bring order to your life. In this challenge, you will build this simple to-do list application that has a React frontend, Express backend, and a MongoDB database. This frontend and backend of the app run in Fargate with an Application Load Balancer (ALB) in front and the database is running in a MongoDB Atlas cluster. When you're done, we'll send you a fancy piece of swag, just for this Challenge! If you feel so inclined to thank The Pulumipus, please write a blog post or post a quick video about it. Tag us on social media or email us at <a href=mailto:da@pulumi.com>da@pulumi.com</a>, and we will send you the special swag.
+    </p>
+    <h3>Prerequisites</h3>
+    <p>In order to complete this challenge, you'll need a couple things set up in advance.</p>
+    <ul>
+      <li>
+        A <a href="https://app.pulumi.com/signup" target="_blank" rel="noopener noreferrer">Pulumi account</a>
+      </li>
+      <li>
+        The <a href="/docs/get-started/install/" target="_blank" rel="noopener noreferrer">Pulumi CLI</a>
+      </li>
+      <li>
+          <a href="https://www.python.org/downloads/">Python 3.9 or higher</a>
+      </li>
+      <li>
+        An <a href="https://portal.aws.amazon.com/" target="_blank" rel="noopener noreferrer">AWS account</a>
+      </li>
+      <li>
+        The <a href="https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html" target="_blank" rel="noopener noreferrer">AWS CLI</a>
+      </li>
+      <li>
+        A <a href="https://www.mongodb.com/cloud/atlas/signup" target="_blank" rel="noopener noreferrer">MongoDB Cloud account</a>
+      </li>
+      <li>
+        A <a href="https://github.com/signup" target="_blank" rel="noopener noreferrer">GitHub account</a>
+      </li>
+    </ul>
+  </div>
+</div>
+
+### Challenge
+
+#### Step 1. Initialize a project
+
+You will first clone the base application and then the rest of the steps will take you through building the Pulumi program.
+
+```shell
+git clone -b challenge https://github.com/aaronkao/atlas-fargate.git
+```
+
+Next, initialize the stack.
+
+```shell
+pulumi stack init dev
```

Comment:
This doesn't quite work here, as the previous command leaves you at a level above where the Pulumi project file is. We should probably change to `atlas-fargate` after cloning.